docs/building-a-form/available-form-features-and-usage-guidelines.md

```diff
@@ -435,8 +450,8 @@ preSubmitInfo: {
 
 #### Usage guidelines
 
-Right now, the required checkbox is automatically included in all forms. The US Forms System team will refactor this component to make it more customizable. To follow that discussion, subscribe to https://github.com/usds/us-forms-system/issues/53.
+No notice or checkbox appears above the submit button unless a `preSubmitInfo` section is specified. Most applications will want, at minimum, to give some sort of notice to the user before the form is submitted. Although this section is optional, it is highly recommended.
```

Comment:
> No notice or checkbox appears above the submit button unless a `preSubmitInfo` section is specified. Most applications will want, at minimum, to give some sort of notice to the user before the form is submitted. Although this section is optional, it is highly recommended.

Less passive:

"Unless you specify a `preSubmitInfo` section, no notice or checkbox appears above the submit button. Most applications will want, at minimum, to give some sort of notice to the user before they submit the form. Although this section is optional, we recommend you specify it."